### ISSUE TITLE
BO - Checkboxes are not aligned on Merchandise returns page when multistore enabled

### ISSUE THREAD

**Issue by HanaRebaiQA** [PrestaShop/PrestaShop#27558]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

Checkboxes and labels are not aligned on Merchandise returns page when multistore enabled
![image](https://user-images.githubusercontent.com/92912932/151986654-ca12b49c-aafe-40e0-8e1a-6a4aee2afc45.png)

### Expected behavior

Checkboxes will be aligned

### Steps to reproduce

1. BO > CONFIGURE > Shop Parameters > General > Enabled Multistore
2. GO to Advanced Parameters > Multistore > Add a new shop
2. GO to SELL > Customer Service > Merchandise Returns
3. See error 
![image](https://user-images.githubusercontent.com/92912932/151987975-d4f704a4-03f4-450a-b0e5-772994aa82dd.png)

### PrestaShop version(s) where the bug happened

1.7.7.8 & develop & 1.7.8.x

### PHP version(s) where the bug happened

7.3

**Comment by Progi1984** [comment]
@PrestaShop/qa-functional  Sorry. But I think it's the expected behavior for legacy pages.

For example the Customer Thread Pages is identical : 
![image](https://user-images.githubusercontent.com/1533248/152050459-5c6f4b76-2a68-4f65-876c-776d453704e5.png)

@NeOMakinG Are you agree or I miss a thing ?

**Comment by HanaRebaiQA** [comment]
Hello @MatShir What do you think about this issue ?

**Comment by ElodieOS** [comment]
Hi everyone ! I’m taking the issue, it’s work in progress. I will keep you informed 😉

**Comment by ElodieOS** [comment]
Hello everyone! 
Little update on this issue.

The idea of aligning the checkboxes is very good, even from an aesthetic point of view, but there are several problems. 
To align the checkboxes, you would have to align all the labels on the left. This change should be done on all the settings of the BO in multistore but also in standard store to keep consistency. It's a big job. In addition, we may have problems with the different size of the labels and the management of spaces that would impact the reading for the user. 

To sum up, I think that changing the alignment would bring more problems than it would solve at the moment while it is not a big inconvenience for the user.

We are working on a modification of the BO layouts, but this one will not happen right away. However, with the first tracks we have this alignment problem will be solved. 

Do we go to UX check @HanaRebaiQA ?

**Comment by MatShir** [comment]
So can we close the issue then? @ElodieOS

**Comment by ElodieOS** [comment]
Yes I think @MatShir
